### ISSUE TITLE
train epoch problem

### ISSUE THREAD

**Issue by poincarelee** [visinf/irr#48]
hi, 
this might be a simple question about training:
 In .sh file, you set the start_epoch as 160 and 490(two train parts),  may I take it as the continuing training of breakpoint at which training was interrupted or it has any other meaning?
  If I want to use some other pretrained  weights (which may not saved with epoch information), start_epoch would be zero just as regular finetuning process?

**Comment by hurjunhwa** [comment]
Hi, 
This start_epoch is used for finding a learning rate given the input LR schedule and the number of epochs to train. It's irrelevant to how many epochs the loaded checkpoint is trained.
If you want to use your own pretrained weights, you can set start_epoch to be zero and configure your own learning rate schedule and the number of epochs to train.
